packages/core/src/node/messaging/messaging-contribution.ts

```diff
@@ -115,9 +122,47 @@ export class MessagingContribution implements BackendApplicationContribution, Me
                 socket.alive = false;
                 socket.ping();
             });
+            this.httpWebsocketAdapters.forEach((adapter, id) => {
+                if (adapter.alive === false) {
+                    adapter.onclose({ code: 404, reason: 'Connection was closed', target: adapter as unknown as WebSocket, wasClean: true });
```

Comment:
I'm a bit confused by the typing here. Is there a downside to modifying the relevant declarations to `WebSocket / ws | HttpWebsocketAdapter`?